lib/logstash/codecs/cef/timestamp_normalizer.rb

```diff
@@ -74,6 +74,8 @@ def get_locale(spec)
   end
 
   def parse_cef_format_string(value, context_timezone=nil)
+    return nil if value.nil? || value.empty?
```

Comment:
Nit: can we update the ruby doc for `normalize` that it will return `nil` if the `value` is `nil` or empty

Reply:
I ended up pulling this special-case out of the `TimestampNormalizer` and up to the caller.